src/cast_core/src/actors/Loader.cpp

```diff
@@ -131,7 +131,7 @@ Loader::Loader(genny::ActorContext& context, uint thread)
       _totalBulkLoadTimer{context.timer("totalBulkInsertTime", Loader::id())},
       _individualBulkLoadTimer{context.timer("individualBulkInsertTime", Loader::id())},
       _indexBuildTimer{context.timer("indexBuildTime", Loader::id())},
-      _client{context.client()},
+      _client{std::move(context.client())},
```

Comment:
Well, it's mildly complicated. The pool entry is a wrapper around a unique pointer, so if we were given a reference or the like we would have to move. The way the class is returned from context, it has no choice but to move. So in short, it's not necessary but it is a good thing to make clear that we're using move construction and not copy construction.

Reply:
hm ya I agree it's better to be explicit but I'm afraid of putting something like this into the skeleton Actor created by `create-new-actor.sh` because rvalues and `std::move` are relatively "scary" parts of modern C++; I fear they'll cause a newbie to throw up their hands. Feedback from Dan was to reduce the C++ complexity as much as possible. TLDR is I'm fine with this for now but I'm hopeful we can eliminate the need for some things like this by maybe pushing some of the complexity into framework code.